app/views/publishers/vacancies/edit_vacancy_sections/_edit_job_role.html.slim

```diff
@@ -1,19 +1,18 @@
-= render "publishers/vacancies/error_tag", attributes: Publishers::JobListing::JobRoleForm.fields
-
 = render_divs_for_fields(Publishers::JobListing::JobRoleForm)
 
 = render ReviewComponent.new id: "job_role" do |review|
   - review.heading title: t("publishers.vacancies.steps.job_role"),
                    text: t("buttons.change"),
-                   href: organisation_job_build_path(@vacancy.id, :job_role)
+                   href: organisation_job_build_path(vacancy.id, :job_role)
+    = vacancy_review_section_tag(vacancy, :job_role)
 
   - review.body do
     = govuk_summary_list do |summary_list|
       - summary_list.row do |row|
         - row.key text: t("jobs.main_job_role")
-        - row.value text: @vacancy.show_main_job_role || t("jobs.not_defined")
+        - row.value text: vacancy.show_main_job_role || t("jobs.not_defined")
```

Comment:
This kind of correction applies in all the _edit_whatever.html.slim files, it seems! Worth doing a separate PR for all those?